File: examples/modulith-clinical-tool-jpa/src/main/java/io/zenwave360/example/clinicaltool/modules/surveys/service/dtos/OptionTranslationOutput.java
package io.zenwave360.example.clinicaltool.modules.surveys.service.dtos;

import io.zenwave360.example.clinicaltool.modules.surveys.domain.*;
import jakarta.validation.constraints.*;
import java.io.Serializable;
import java.math.*;
import java.time.*;
import java.util.*;

/**
 * OptionTranslationOutput.
 */
@lombok.Getter
@lombok.Setter
public class OptionTranslationOutput implements Serializable {

    @NotNull
    @Size(max = 254)
    private String name;

    @NotNull
    private String text;
}

File: examples/modulith-klinical-tool-jpa/src/main/kotlin/io/zenwave360/example/clinicaltool/modules/surveys/service/dtos/OptionTranslationOutput.kt
package io.zenwave360.example.clinicaltool.modules.surveys.service.dtos

import io.zenwave360.example.clinicaltool.modules.surveys.domain.*
import java.io.Serializable
import java.math.*
import java.time.*
import jakarta.validation.constraints.*

/**
 * OptionTranslationOutput.
 */
data class OptionTranslationOutput(

    @NotNull @Size(max = 254)
    val name: String?  = null,

    @NotNull
    val text: String?  = null

)  : Serializable {




}
